torch/jit/_serialization.py

```diff
@@ -149,10 +149,10 @@ def load(f, map_location=None, _extra_files=None, _restore_shapes=False):
         os.remove("scriptmodule.pt")
     """
     if isinstance(f, (str, os.PathLike)):
```

Comment:
I think was also originally supposed to accept bytes, and might have been killed in some refactor. Hence the type:ignore

Reply:
Yes, that would also have been my guess. In any case, there is currently no type error here.